app/src/main/java/com/sapuseven/untis/activities/MainActivity.kt

```diff
@@ -10,9 +10,7 @@ import android.net.Uri
 import android.os.Bundle
 import android.os.Handler
 import android.os.Looper
-import android.view.MenuItem
-import android.view.View
-import android.view.ViewGroup
+import android.view.*
```

Comment:
Any reason for that wildcard import?

Reply:
It was automatically created by Android Studio

Reply:
Alright, in this case it's acceptable.